src/network/RTypeClient.hpp

```diff
@@ -0,0 +1,64 @@
+/*
+** EPITECH PROJECT, 2024
+** RType
+** File description:
+** RTypeClient
+*/
+
+#ifndef RTYPE_CLIENT_HPP_
+    #define RTYPE_CLIENT_HPP_
+    #include <unordered_map>
+    #include "UDPConnection.hpp"
+    #include "GameEngine/GameEngine.hpp"
+
+class RTypeClient : public UDPConnection
```

Comment:
Can you add a doxygen documentation to this class ? So that we know what it does

Reply:
In progress

Reply:
Done